cirq-core/cirq/devices/named_topologies.py

```diff
@@ -142,6 +142,13 @@ def draw(self, ax=None, tilted: bool = True, **kwargs) -> Dict[Any, Tuple[int, i
         g2 = nx.relabel_nodes(self.graph, {n: (n, 1) for n in self.graph.nodes})
         return draw_gridlike(g2, ax=ax, tilted=tilted, **kwargs)
 
+    def nodes_to_linequbits(self, offset=0) -> Dict[Tuple[int, int], 'cirq.LineQubit']:
+        """Return a mapping from graph nodes to `cirq.LineQubit`
+        Args:
+            offset: Offset LineQubits by this amount.
+        """
+        return {(r, 1): LineQubit(r) + offset for r in self.graph.nodes}
```

Comment:
The draw method outputs 2-int tuples even for the LineTopology. Given this, should the nodes be represented as integers or (x, y) coordinates?
```
topology = LineTopology(5)
print(topology.draw(tilted = False))
```

Outputs:

`{(0, 1): (1, 1), (1, 1): (2, 0), (2, 1): (3, -1), (3, 1): (4, -2), (4, 1): (5, -3)}`
<img width="313" alt="Screenshot 2022-08-10 at 6 51 36 AM" src="https://user-images.githubusercontent.com/9400738/183789769-bd96bfab-4062-4342-9df6-759ddc24ab64.png">

Reply:
The issue https://github.com/quantumlib/Cirq/issues/4710#issue-1061951421 that this PR is for states that nodes_to_linequbits method is needed for relabeling nodes with
```
routing_graph = nx.relabel_nodes(topology.graph, topology.nodes_to_linequbits())
```
That does not work if it returns a dictionary with a pair-tuple keys instead of integers that are in the graph.

The issue does not say anything about drawing the graph or using nodes_to_linequbits for such.

Reply:
Ah, got it, thanks! Made the fixes.